runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/skills/MiningAction.java

```diff
@@ -64,13 +64,21 @@ public boolean isMembers(final ItemManager itemManager)
 	SANDSTONE_5KG(ItemID.SANDSTONE_5KG, 35, 50),
 	SANDSTONE_10KG(ItemID.SANDSTONE_10KG, 35, 60),
 	DENSE_ESSENCE_BLOCK(ItemID.DENSE_ESSENCE_BLOCK, 38, 12),
-	GOLD_ORE(ItemID.GOLD_ORE, 40, 65),
 	GEM_ROCKS(ItemID.UNCUT_RED_TOPAZ, 40, 65)
+			{
+				@Override
+				public String getName(final ItemManager itemManager)
+				{
+					return "Gem rocks";
+				}
+			},
```

Comment:
FYI I know this is how intellij's auto-format indents this block, but we'd prefer the block start at the same indentation as the enum definition, eg.

```java
	ENUM_ENTRY()
	{
		@Override
		/ * ... */
	},
```

Reply:
Is there a setting for intellij to change this? I completely agree with this being more readable.